app/operations/organizations/create.rb

```diff
@@ -0,0 +1,21 @@
+module Organizations
+  class Create < Operation
+    string :name
+    string :display_name
+    string :primary_language
+
+    # Organization Contents Fields
+    string :title
+    string :description
+    string :introduction, default: ''
+
+    def execute
+      Organization.transaction do
+        organization = Organization.new owner: api_user.user, name: name, display_name: display_name, primary_language: primary_language
+        organization.organization_contents.build title: title, description: description, introduction: introduction, language: primary_language
+        organization.save!
+        organization
```

Comment:
I think so, since #save! returns a boolean. The return object should be an org (or an array thereof) to pass to `created_resource_response`.